### DISCUSSION TITLE
arcticfoxes.net Matrix instance is down (and therefore the PrivSec.dev:arcticfoxes.net server too)

### DISCUSSION THREAD

**Discussion by ghost** [PrivSec-dev/privsec.dev#145] [Q&A] [answered]
Hi,
The Privsec.dev matrix room can't be reached anymore since the arcticfoxes.net server is down as per their maintainance note. Are there any plans to move the Privsec.dev room to some other server?

**Comment by wj25czxj47bu6q** [comment, marked as answer]
Matrix rooms are not centralized. You may join the rooms via the following links:

- [PrivSec.dev — Practical Privacy & Security](https://matrix.to/#/!frWBWOPyriepLexnsA:arcticfoxes.net?via=tchncs.de&via=jameskitt616.one&via=matrix.org)
- [PrivSec.dev — Off&#8209;topic](https://matrix.to/#/!VKDlypqKDjMhDHKPlf:arcticfoxes.net?via=tchncs.de&via=jameskitt616.one&via=matrix.org)
- [PrivSec.dev Space](https://matrix.to/#/!KnrDAAJRrUsPkFLYId:arcticfoxes.net?via=tchncs.de&via=jameskitt616.one&via=matrix.org)
